File: app/src/main/java/com/abona_erp/driver/app/data/model/ResultOfAction.java
package com.abona_erp.driver.app.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ResultOfAction {
  
  @SerializedName("IsSuccess")
  @Expose
  private boolean isSuccess;
  
  @SerializedName("IsException")
  @Expose
  private boolean isException;
  
  @SerializedName("Text")
  @Expose
  private String text;
  
  @SerializedName("CommunicationItem")
  @Expose
  private CommItem commItem;
  
  
  public boolean getIsSuccess() {
    return isSuccess;
  }
  
  public void setIsSuccess(boolean isSuccess) {
    this.isSuccess = isSuccess;
  }
  
  public boolean getIsException() {
    return isException;
  }
  
  public void setIsException(boolean isException) {
    this.isException = isException;
  }
  
  public String getText() {
    return text;
  }
  
  public void setText(String text) {
    this.text = text;
  }
  
  public CommItem getCommItem() {
    return commItem;
  }
  
  public void setCommItem(CommItem commItem) {
    this.commItem = commItem;
  }
}

File: app/src/main/java/com/abona_erp/driver/app/data/model/ResultOfAction.kt
package com.abona_erp.driver.app.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResultOfAction {
    @SerializedName("IsSuccess")
    var isSuccess = false

    @SerializedName("IsException")
    var isException = false

    @SerializedName("Text")
    var text: String? = null

    @SerializedName("CommunicationItem")
    var commItem: CommItem? = null

    @SerializedName("AllTask")
    var allTask: List<TaskItem>? = null

    @SerializedName("DelayReasons")
    var delayReasonItems: List<DelayReasonItem>? = null

    override fun toString(): String {
        return "ResultOfAction{" +
                "isSuccess=" + isSuccess +
                ", isException=" + isException +
                ", text='" + text + '\'' +
                ", commItem=" + commItem +
                ", allTask=" + allTask +
                ", delayReasonItems=" + delayReasonItems +
                '}'
    }
}